app/controllers/users_controller.rb

```diff
@@ -0,0 +1,17 @@
+class UsersController < ApplicationController
+before_action :require_login
+
+  def show
+    @user = current_user
+    @tutorials = @user.tutorials
+  end
+
+  private
+  def require_login
```

Comment:
Again, Devise has a helper for this, but if you do decide to keep your own implementation, move this method in to the application controller so it doesn't need to be repeated in every controller you want to limit access to.